src/main/java/com/nulabinc/zxcvbn/matchers/SpatialMatcher.java

```diff
@@ -46,15 +49,15 @@ public SpatialMatcher() {
                 shiftedCount = 0;
             }
             final Map<Character, List<String>> graph = keyboard.getAdjacencyGraph();
-            while(true) {
-                Character prevChar = password.charAt(j-1);
+            while (true) {
+                Character prevChar = password.charAt(j - 1);
                 boolean found = false;
-                int foundDirection = -1;
+                int foundDirection;
```

Comment:
Thanks! The initial value of `foundDirection` doesn't have to be `-1`. It is always initialized with `curDirection`.  👍